docs/api/graphql/getting-started.md

```diff
@@ -17,54 +17,40 @@ We'll reiterate a few important points before proceeding:
-To do so, include the `X-DataHub-Actor` header with an Authorized Corp User URN as the value in your request. Because anyone is able to set the value of this header, we recommend using this endpoint only in trusted environments, either by administrators themselves or programs that they own directly. 
-   
-2. **Frontend Proxy**: The DataHub Frontend Proxy Service (frontend) is a basic web server & reverse proxy to the Metadata Service. As such, the 
-GraphQL endpoint is also available for query wherever the Frontend Proxy is deployed. In local deployments, this is typically `http://localhost:9002/api/v2/graphql`. By default,
-the Frontend Proxy *does* have Session Cookie-based Authentication via the PLAY_SESSION cookie set at DataHub UI login time. This means
-that if a request does not have a valid PLAY_SESSION cookie obtained via logging into the DataHub UI, the request will be rejected. To use this API in an untrusted environment,
-you'd need to a) log into DataHub, b) extract the PLAY_SESSION cookie that is set on login, and c) provide this Cookie in your HTTP headers when
-calling the endpoint.
-   
+#### Authentication + Authorization
 
-### Querying the Endpoint
+In general, you'll need to provide an [Access Token](../../authentication/personal-access-tokens.md) when querying the GraphQL by
+providing an `Authorization` header containing a `Bearer` token. The header should take the following format:
 
-There are a few options when it comes to querying the GraphQL endpoint. The recommendation on which to use varies by use case.
+```bash
+Authorization: Bearer <access-token>
+```
 
-**Testing**: [Postman](https://learning.postman.com/docs/sending-requests/supported-api-frameworks/graphql/), GraphQL Explorer (described below), CURL
+Authorization for actions exposed by the GraphQL endpoint will be performed based  on the actor making the request.
+For Personal Access Tokens, the token will carry the user's privileges. 
 
-**Production**: GraphQL [Client SDK](https://graphql.org/code/) for the language of your choice, or a basic HTTP client.
-   
-> Important: The DataHub GraphQL endpoint only supports POST requests at this time. It does not support GET requests. If this is something
-> you need, let us know on Slack!
+> Notice: The DataHub GraphQL endpoint only supports POST requests at this time.
 
 ### On the Horizon
 
-- **Service Access Tokens**: In the near future, the DataHub team intends to introduce service users, which will provide a way to generate and use API access
+- **Service Tokens**: In the near future, the DataHub team intends to introduce service users, which will provide a way to generate and use API access
 tokens when querying both the Frontend Proxy Server and the Metadata Service. If you're interested in contributing, please [reach out on our Slack](https://datahubspace.slack.com/join/shared_invite/zt-nx7i0dj7-I3IJYC551vpnvvjIaNRRGw#/shared-invite/email).
```

Comment:
this link may not be valid. I think `https://slack.datahubproject.io` is the evergreen link.

Reply:
Updated!